tests/postgres/test-attr.rs

```diff
@@ -42,7 +42,11 @@ async fn it_gets_users(pool: PgPool) -> sqlx::Result<()> {
     Ok(())
 }
 
-#[sqlx::test(migrations = "tests/postgres/migrations", fixtures("users", "posts"))]
+// This should apply migrations and then `../fixtures-postgres/users.sql` and `../fixtures-postgres/posts.sql`
+#[sqlx::test(
+    migrations = "tests/postgres/migrations",
+    fixtures_path("../fixtures-postgres/users", "../fixtures-postgres/posts")
+)]
```

Comment:
I would really prefer if you use a new test instead of co-opting an existing one. Even just making a copy of this one would be fine.

Reply:
Sure, will work on that asap.